src/tests/Navabar.test.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import Navbar from '../components/Navbar';
+
```

Comment:
- [ ] Hi, I will advise you to test more components of the App, because the requirements say that the App must have unit tests for several components of the app.

- [ ] Can you add integration tests using React Testing Library, it is the project requirement. 
 For this one, you can perform a mocking data fetching and Redux Store connection.